README.md

```diff
@@ -146,6 +147,9 @@ Apply any view style props to the container rather than use the default look (e.
 #### `separatorStyle`: (optional)
 Modify the look of the separators rather than use the default look.
 
+#### `useModal`: (optional)
+Wrap the ActionSheet with a Modal, in order to show in front of other Modals that were already opened ([issue reference](https://github.com/expo/react-native-action-sheet/issues/164)).
+
```

Comment:
I'm not the best English speaker, so, @bradbumbalough please double check this text